skfda/ml/classification/_parametrized_functional_qda.py

```diff
@@ -0,0 +1,229 @@
+from __future__ import annotations
+
+import numpy as np
+from GPy.kern import Kern
+from GPy.models import GPRegression
+from scipy.linalg import logm
+from sklearn.base import BaseEstimator, ClassifierMixin
+from sklearn.utils.validation import check_is_fitted
+
+from ..._utils import _classifier_get_classes
+from ...representation import FDataGrid
+
+
+class ParametrizedFunctionalQDA(
+    BaseEstimator,  # type: ignore
+    ClassifierMixin,  # type: ignore
+):
+    """Parametrized functional quadratic discriminant analysis.
+
+    This classifier is based on the assumption that the data is part
```

Comment:
I think we should add a warning in the documentation that this classifier is experimental, as it does not come from a peer-published paper. Please bring this topic to next meeting.